tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.h

```diff
@@ -145,7 +146,8 @@ class IrEmitterUnnested : public IrEmitter {
 
   IrEmitterUnnested(const HloModuleConfig& hlo_module_config,
                     const HloComputation* hlo_computation,
-                    IrEmitterContext* ir_emitter_context);
+                    IrEmitterContext* ir_emitter_context,
+                    llvm_ir::LLVMTargetFeatures* llvm_target_features = nullptr);
```

Comment:
In case `IrEmitterUnnested` is constructed without an `llvm_target_features`, `llvm_target_features_` field in `IrEmitter` would become a garbage reference.

Reply:
Provided targetIRBuilder at instances where IrEmitterUnnested is constructed